specutils/wcs/wcs_wrapper.py

```diff
@@ -0,0 +1,64 @@
+import gwcs
+import numpy as np
+from astropy.modeling.tabular import Tabular1D
+import astropy.units as u
+from gwcs import coordinate_frames as cf
+
+from .adapters import *
+from .wcs_adapter import WCSAdapter
+
+__all__ = ['WCSWrapper']
+
+
+class WCSWrapper:
```

Comment:
do we need `class WCSWrapper(object):`

Reply:
In python 3, `class WCSWrapper` and `class WCSWrapper` are equivalent. But you're correct for the python 2 case: `object` should be explicitly inherited from.

Reply:
I suppose we're fine with not supporting py2 from the start?